docs/ref/contrib/admin/index.txt

```diff
@@ -1887,11 +1887,15 @@ templates used by the :class:`ModelAdmin` views:
     Override this method to customize the lookups permitted for your
     :class:`~django.contrib.admin.ModelAdmin` subclass.
 
-.. method:: ModelAdmin.has_add_permission(request)
+.. method:: ModelAdmin.has_add_permission(request, obj=None)
 
     Should return ``True`` if adding an object is permitted, ``False``
     otherwise.
 
+    .. versionchanged:: 2.1
+
+    The ``obj`` argument was added.
```

Comment:
What does the `obj` argument represent at the `ModelAdmin` level? (At the inline level it's the parent... but here?)